### DISCUSSION TITLE
Show 0 value on axis

### DISCUSSION THREAD

**Discussion by mutheusalmeida** [FormidableLabs/victory#2641] [General]
Hey guys. How can I show $0k on the Y-axis of my chat? I tried using the domain property but it didn't work. Anybody knows?

![Captura de Tela (7)](https://github.com/FormidableLabs/victory/assets/34585988/e7db4e80-ede9-4bf1-8e75-8d71d585431e)

**Comment by nambiarn** [comment]
[Here's a link to the documentation](https://formidable.com/open-source/victory/docs/faq/#how-can-i-change-the-position-of-my-axis) where it calls out how 0 is turned off by default. 

>TLDR; To change this behavior, set the crossAxis prop to false.
